oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollector.java

```diff
@@ -31,6 +32,34 @@ public interface FullGCStatsCollector {
      */
     void documentRead();
 
+    /**
+     * Total No. of properties deleted during a given GC phase
+     * @param mode GC phase
+     * @param numProps no. of properties deleted in current cycle
+     */
+    void collectedPropertiesDeleted(GCPhase mode, long numProps);
+
+    /**
+     * Total No. of documents updated during a given GC phase
+     * @param mode GC phase
+     * @param numCommits no. of unmerged branch commits in current cycle
+     */
+    void collectedUnmergedBranchCommits(GCPhase mode, long numCommits);
```

Comment:
same as above. Duplicate of `void unmergedBranchCommitsDeleted(long numCommits);`.